json-rpc-core/build.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
+    id "java"
+}
+
+group 'com.amazonaws.blox'
+version '0.1-SNAPSHOT'
+
+sourceCompatibility = 1.8
+
+dependencies {
+    compile "com.github.briandilley.jsonrpc4j:jsonrpc4j:+"
+    compile "org.projectlombok:lombok"
+
+    testCompile "org.slf4j:slf4j-simple"
+    testCompile "org.projectlombok:lombok"
```

Comment:
`testCompileOnly`?

Reply:
Hm, I thought I tried that without IDE success. Will double check.